modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/resolver/ObjectBuilderValueResolver.java

```diff
@@ -77,4 +80,13 @@ public Object getParameterValue(String parameterName) throws ValueResolvingExcep
       throw new ValueResolvingException(format("Unable to resolve value for parameter '%s'", parameterName));
     }
   }
+
+  @Override
+  public Map<String, ValueResolver<? extends Object>> getParameters() {
+    if (builder instanceof ParameterValueResolver) {
+      return ((ParameterValueResolver) builder).getParameters();
+    } else {
+      return new HashMap<>();
+    }
```

Comment:
Return empty HashMap or throw exception like the method above?

Reply:
return `Collections.emptyMap()`

Reply:
done